spec/config/initializers/ab_tests_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+describe AbTests do
+  def reload_ab_test_initializer!
+    # undefine the AB tests instances so we can re-initialize them with different config values
+    AbTests.constants.each do |const_name|
+      AbTests.class_eval { remove_const(const_name) }
+    end
+    load Rails.root.join('config', 'initializers', 'ab_tests.rb').to_s
+  end
+
+  describe '::NATIVE_CAMERA' do
+    let(:percent) { 30 }
+
+    before do
+      allow(IdentityConfig.store).to receive(:idv_native_camera_a_b_testing_enabled).
+        and_return(true)
+      allow(IdentityConfig.store).to receive(:idv_native_camera_a_b_testing_percent).
+        and_return(percent)
+
+      reload_ab_test_initializer!
+    end
+
+    after do
+      allow(IdentityConfig.store).to receive(:idv_native_camera_a_b_testing_enabled).
+        and_call_original
+      allow(IdentityConfig.store).to receive(:idv_native_camera_a_b_testing_percent).
+        and_call_original
+
+      reload_ab_test_initializer!
+    end
```

Comment:
This is the one downside to moving these to an initializer, when we want to test the parameters, we have to resort to haxx to reload the objects and reload the initializer

However, I think there are a few reasons why maybe we shouldn't need to reload
- The AbTestBucket class tests the behavior and bucketing, most tests should be assigning buckets directly
- I think we can reduce the number of config variables and reduce our complexity overall [see comment](https://github.com/18F/identity-idp/pull/6994#discussion_r975891218)